test/doc_comments_test.dart

```diff
@@ -0,0 +1,199 @@
+// Copyright 2018 Google Inc. Use of this source code is governed by an
+// MIT-style license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import 'package:sass/src/ast/sass.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('documentation comments', () {
+    group('SCSS syntax:', () {
+      test('variable declarations are found', () {
+        final contents = r'''
+            /// Results my vary.
+            $vary: 5.16em;''';
+        final stylesheet = Stylesheet.parseScss(contents);
+        final variable = stylesheet.children
+                .firstWhere((child) => child is VariableDeclaration)
```

Comment:
```suggestion
                .whereType<VariableDeclaration>().first
```

This will let you avoid the awkward type duplication.

Reply:
Done.